stdlib/public/CMakeLists.txt

```diff
@@ -72,16 +72,16 @@ endif()
 # SWIFT_ENABLE_TENSORFLOW
 if(SWIFT_BUILD_STDLIB)
   add_subdirectory(Python)
-endif()
 
-# SWIFT_ENABLE_TENSORFLOW
-# Note: Python should be built before TensorFlow because Python is a
-# dependency for `numpy.ndarray` bridging to `ShapedArray`/`Tensor`.
-if(SWIFT_BUILD_STDLIB AND SWIFT_ENABLE_TENSORFLOW)
-  # TODO: Add TensorFlow support for iOS/Raspberry Pi.
-  add_subdirectory(CTensorFlow)
-  add_subdirectory(TensorFlow)
+  # Note: Python should be built before TensorFlow because Python is a
+  # dependency for `numpy.ndarray` bridging to `ShapedArray`/`Tensor`.
+  if(SWIFT_ENABLE_TENSORFLOW)
+    # TODO: Add TensorFlow support for iOS/Raspberry Pi.
+    add_subdirectory(CTensorFlow)
+    add_subdirectory(TensorFlow)
+  endif()
 endif()
+# SWIFT_ENABLE_TENSORFLOW
```

Comment:
Did you mean to use an `END` marker here?
```suggestion
# SWIFT_ENABLE_TENSORFLOW END
```

Reply:
I did indeed mean to use the `SWIFT_ENABLE_TENSORFLOW END`.  Thanks